api/app/controllers/mno_enterprise/jpi/v1/admin/impac/dashboard_templates_controller.rb

```diff
@@ -1,6 +1,8 @@
 module MnoEnterprise
   class Jpi::V1::Admin::Impac::DashboardTemplatesController < Jpi::V1::Admin::BaseResourceController
 
+    before_action :load_organizations, except: [:destroy]
```

Comment:
Where's `@organizations` used? Only in the view?
With the `before_action` you're loading organizations all the time, even if the `find_one!` or `save!` is going to fail

Reply:
True. I'll update it.